src/grants.ts

```diff
@@ -1,3 +1,10 @@
+export enum TrackSource {
+  CAMERA = 'camera',
+  MICROPHONE = 'microphone',
+  SCREEN_SHARE = 'screen_share',
+  SCREEN_SHARE_AUDIO = 'screen_share_audio',
```

Comment:
how should we handle `unknown` sources? 
list them explicitly here as well?

Reply:
I'm not sure what the best approach for unknown is.. in terms of specifying you'd only want users to publish a track without a source identification?

perhaps we can leave it off until someone thinks of a use case for it. easier to add later on than it is to remove.